ui/app/models/service-fragment.js

```diff
@@ -0,0 +1,11 @@
+import { attr } from '@ember-data/model';
+import Fragment from 'ember-data-model-fragments/fragment';
+import { fragment } from 'ember-data-model-fragments/attributes';
+
+export default class Service extends Fragment {
+  @attr('string') name;
+  @attr('string') portLabel;
+  @attr() tags;
+  @attr('string') onUpdate;
+  @fragment('consul-connect') connect;
+}
```

Comment:
This is consumed by `TaskGroup` and in the associated `Allocation`, `TaskGroup` and `Task` pages. I'm noticing that CI detected 19 failures and error messages failing on tests like:

`allocation detail: if stopping or restarting fails, an error message is shown`.

Any idea why these tests are failing CI?

Reply:
Yep! Had forgotten to rename the mirage model: https://github.com/hashicorp/nomad/pull/13856/commits/3cf40b711a3615384765e47e9fa847aa9d4755f2

now settled